js/languages/en-US.json

```diff
@@ -27,6 +27,13 @@
   "serverConnect": {
     "defaultServerName": "Default Server"
   },
+  "mainNav": {
+    "followers": "Followers",
```

Comment:
These are stings in the User Page and therefore should be embedded in the userPage key.